interfaces/ResizeObserver.idl

```diff
@@ -0,0 +1,25 @@
+[Constructor(ResizeObserverCallback callback),
+    Exposed=Window]
+interface ResizeObserver {
+    void observe(Element target);
+    void unobserve(Element target);
+    void disconnect();
+};
+
+callback ResizeObserverCallback = void (sequence<ResizeObserverEntry> entries, ResizeObserver observer)
```

Comment:
Sent https://github.com/WICG/ResizeObserver/pull/56 to fix a missing semicolon here, will add it manually too.

Reply:
Added manually for now.